README.md

```diff
@@ -108,19 +108,19 @@ Use `make test` for running the unit(integration) tests:
 make test
 ```
 
-If you wish to run a single test, just edit the `Dockerfile.development`:
+If you wish to run a single test, just edit the `Dockerfile`:
 
 ```diff
-diff --git a/Dockerfile.development b/Dockerfile.development
-index 81fbb349c014..431f07b59917 100644
---- a/Dockerfile.development
-+++ b/Dockerfile.development
-@@ -4,4 +4,4 @@ FROM repo-resource:latest
+diff --git a/Dockerfile b/Dockerfile
+index f02c512..031008e 100644
+--- a/Dockerfile
++++ b/Dockerfile
+@@ -28,6 +28,6 @@ FROM repo-resource AS tests
  COPY repo_resource /root/repo_resource
  COPY development/ssh/ /root/development/ssh/
  WORKDIR /root/
 -CMD python -m unittest
-+CMD python -m unittest repo_resource.test_check.TestCheck.test_branch_defaults_to_HEAD
++RUN python -m unittest repo_resource.test_check.TestCheck.test_branch_defaults_to_HEAD
```

Comment:
Why is there a `RUN` here instead of a `CMD` ?

This diff block should be:
```diff
diff --git a/Dockerfile b/Dockerfile
index f02c512fc2c6..40c94ec1b717 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -28,6 +28,6 @@ FROM repo-resource AS tests
 COPY repo_resource /root/repo_resource
 COPY development/ssh/ /root/development/ssh/
 WORKDIR /root/
-CMD python -m unittest
+CMD python -m unittest unittest repo_resource.test_check.TestCheck.test_branch_defaults_to_HEAD

 FROM repo-resource
```